data_structures/linked_list/circular_linked_list.py

```diff
@@ -0,0 +1,224 @@
+import typing
```

Comment:
Just a thought, I generally import all standard library directly by their module/package name in order to avoid namespace collision with 3P var/module/package/class etc. Is the above requested change only for readability, I agree if it is

Reply:
Done for readability.  Follows the pattern in the examples in the __typing__ module documentation.

Reply:
Thanks, for the learning :)